client/src/components/ErrorMessage.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Box, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+
+const useStyles = makeStyles({
+  container: {
+    backgroundColor: "red",
+    marginLeft: "20%",
+    marginRight: "20%",
+    marginTop: "2vh",
+    marginBottom: "2vh",
+    borderRadius: "1vh",
+    boxShadow: "0.1vh 0.1vh 0.5vh grey"
+  },
+  text: {
+    color: "white",
+    lineHeight: "5vh",
+    fontWeight: "800"
+  }
+});
+
+const ErrorMessage = props => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.container}>
+      <Typography className={classes.text}>{props.message}</Typography>
+    </Box>
```

Comment:
I like that you are creating re-usable components! Looking forward to how this looks :) Wondering how it'll compare to using a snackbar component instead

Reply:
I tried using snackbars initially but I couldn't quite figure it out. From the example on the material ui page they were using an alert from @ material-ui/lab, but it didn't work for me.

Reply:
for alerts you'd have to install @material-ui/lab with npm, but I don't think we need to go that right. You may want to play around more with snackbars to see how they work, they will be very useful